pkgs/development/libraries/readstat/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoreconfHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "readstat";
+  version = "1.1.8";
+
+  src = fetchFromGitHub {
+    owner = "WizardMac";
+    repo = "ReadStat";
+    rev = "v${version}";
+    sha256 = "sha256-xB8GPdasqmbVtMAvirsiGyv/ncTRvRo2GdYHWAimBOQ=";
+  };
+
+  nativeBuildInputs = [
+    autoreconfHook
```

Comment:
Why not compile from source?

Reply:
readstat seems to have already been added to nixpkgs